libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/RestClient.swift

```diff
@@ -136,10 +136,10 @@ extension RestClient {
     /// - See   [Composite](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_composite.htm).
     public func send(compositeRequest: CompositeRequest, _ completionBlock: @escaping (Result<CompositeResponse, RestClientError>) -> Void) {
         compositeRequest.parseResponse = false
-        __sendCompositeRESTRequest(compositeRequest, fail: { (error, urlResponse) in
+        __send(compositeRequest, failureBlock: { (response, error, urlResponse) in
```

Comment:
Should the RestClientError also capture the response?

Reply:
Wouldn't the caller have access to both?

Reply:
If you use RestClient.swift, the completion block in case of error only gets Result.failure(apiError).

Reply:
Yes, I just realized. I focused on the `Objective C` side of things ;-)